web/src/overlays.ts

```diff
@@ -49,6 +49,14 @@ function call_hooks(func_list: Hook[]): void {
     }
 }
 
+export function disable_scrolling(): void {
+    $("html").css({"overflow-y": "hidden"});
+}
+
+function enable_scrolling(): void {
+    $("html").css({"overflow-y": "scroll"});
+}
```

Comment:
@karlstolley this seems to have worked surprisingly well, I am not sure what we were doing wrong before. I might have confused results with my attempt to move scrollbar to `body`.

Reply:
You just needed @andersk's pronouncement that that was the way to go :) (I blame myself for not having finished my coffee yet this morning.) Either way, I'm super happy for you that this worked out.